cmd/nodecmd/stop.go

```diff
@@ -39,6 +42,7 @@ Note that a stopped node may still incur cloud server storage fees.`,
 	}
 	cmd.Flags().BoolVar(&authorizeAccess, "authorize-access", false, "authorize CLI to release cloud resources")
 	cmd.Flags().BoolVar(&authorizeRemove, "authorize-remove", false, "authorize CLI to remove all local files related to cloud nodes")
+	cmd.Flags().BoolVar(&ignoreAlreadyStopped, "ignore-already-stopped-nodes", false, "do not consider already stopped nodes as an error, and remove cluster info anyway")
```

Comment:
this should be default behavior. If node is stopped and user want all nodes in the cluster to be stopped cli should just silently carry on